packages/paste-core/components/side-modal/src/SideModal.tsx

```diff
@@ -37,20 +16,21 @@ const getAnimationStates = (): any => ({
   leave: {opacity: 0, transform: `translateX(100%)`},
   // https://www.react-spring.dev/docs/advanced/config
   config: {
-    mass: 0.3,
-    tension: 288,
-    friction: 15,
+    mass: 0.5,
+    tension: 220,
+    friction: 20,
   },
 });
 
 export interface SideModalProps extends React.HTMLAttributes<HTMLDivElement> {
   children: React.ReactNode;
   'aria-label': string;
   element?: BoxProps['element'];
+  hideOnEsc?: boolean;
 }
 
 export const SideModal = React.forwardRef<HTMLDivElement, SideModalProps>(
-  ({children, element = 'SIDE_MODAL', ...props}, ref) => {
+  ({children, element = 'SIDE_MODAL', hideOnEsc = false, ...props}, ref) => {
```

Comment:
issue: technically, we should do dialogs the other way round. By default, they close on escape, but giving the option to override that.